### DISCUSSION TITLE
incorrect display

### DISCUSSION THREAD

**Discussion by Freed-Wu** [inducer/pudb#577] [Troubleshooting]
![screen-2022-12-08-21-01-49](https://user-images.githubusercontent.com/32936898/206453607-d7a016e1-c337-4841-8a68-a66395ab1e5d.jpg)

```python
from rich.progress import track
for i in track(range(3)):
    breakpoint()
    print(i)
```

**Comment by mvanderkamp** [comment]
Looks like you did something to your terminal to get it to print escape sequences instead of interpreting them.

**Reply by Freed-Wu** [reply]
Not me :cry: should be related to rich
